zio/zjsonio/types.go

```diff
@@ -185,6 +278,6 @@ func (d decoder) decodeTypeMap(zctx *zed.Context, m *astzed.TypeMap) (*zed.TypeM
 	return zctx.LookupTypeMap(keyType, valType), nil
 }
 
-func (d decoder) decodeTypeEnum(zctx *zed.Context, enum *astzed.TypeEnum) (*zed.TypeEnum, error) {
+func (d decoder) decodeTypeEnum(zctx *zed.Context, enum *Enum) (*zed.TypeEnum, error) {
 	return nil, errors.New("TBD: issue #2508")
```

Comment:
Is this the right time to fix this and enable `zio/zjsonio/ztests/enum.yaml`?

Reply:
yes will do